File: assignment1/PizzaPricer.py
#importing math package
import math

#Declering litrealas and assigning them with instructions
diameter=int(input("Pizza Pricer\n Enter diametermeter of pizza (inches): ")) #integer
dollers=float(input("Enter cost of pizza (dollars) : ")) # float

#process to compute the conditions
#converting dollers to cents
cents=int(dollers*100) #Integer 

#calculating area
area=math.pi*diameter*diameter*0.25 # float

#calculating how many people can eat pizza 
people=int(area/50) #Integer

#calculating cost in cents
centsPerSquareInch=cents/area #float

#calculting dollers need to pay accordingly with tip
cost=dollers/people #float
tip=(15*cost)/100 #float
costPerPerson=cost+tip #float

#Displaying the reqested output
print("\n This pizza",people,
    "people\n  cents/square inch\t =",centsPerSquareInch,
    "\n  cost/person\t =",costPerPerson,"dollers (includes a 15% tip)")

File: assignment1/PizzaPricer.java
package assignment1;

import java.util.Scanner;

public class PizzaPricer {
    public static void main(String[] args) {
        Scanner in=new Scanner(System.in);
        
        int diameter=0;
        int cents;
        int people;
        double dollers;
        double area;
        double centsPerSquareInch;
        double tip;
        double cost;
        double costPerPerson;


        System.out.println("Pizza Pricer\n Enter diametermeter of pizza (inches): ");
        diameter=in.nextInt();
        System.out.println("Enter cost of pizza (dollars) : ");
        dollers=in.nextDouble();

        cents=(int)(dollers*100);
        area=Math.PI*diameter*diameter*0.25;
        people=(int)area/50;
        centsPerSquareInch=cents/area;
        cost=dollers/people;
        tip=0.15*cost;
        costPerPerson=cost+tip;

        System.out.println("\n This pizza " +people+" people");
        System.out.println("  cents/square inch\t ="+ centsPerSquareInch);
        System.out.println("  cost/person\t = "+ costPerPerson+ " dollers (includes a 15% tip)");

        in.close();
    }
}
